src/test/java/hudson/plugins/git/browser/TFS2013GitRepositoryBrowserTest.java

```diff
@@ -15,31 +16,30 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertEquals;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class TFS2013GitRepositoryBrowserTest {
 
-    private static final String projectName = "fisheyeProjectName";
-
-    private final String repoUrl;
-    private final GitChangeSetSample sample;
-
-    public TFS2013GitRepositoryBrowserTest() {
-        this.repoUrl = "http://tfs/tfs/project/_git/repo";
-        sample = new GitChangeSetSample(false);
+    private static final String repoUrl = "http://tfs/tfs/project/_git/repo";
+    private static final GitChangeSetSample sample = new GitChangeSetSample(false);
+    
+    @BeforeClass
+    public static void testSetUp() {
 
         GitSCM scm = new GitSCM(
-                Collections.singletonList(new UserRemoteConfig(repoUrl, null, null, null)),
-                new ArrayList<>(),
-                null, JGitTool.MAGIC_EXENAME,
-                Collections.<GitSCMExtension>emptyList());
-        
+            Collections.singletonList(new UserRemoteConfig(repoUrl, null, null, null)),
+            new ArrayList<>(),
+            null, JGitTool.MAGIC_EXENAME,
+            Collections.<GitSCMExtension>emptyList());
+
```

Comment:
Generally ask that changes of pure white space not be made because there is a large backlog of closed but potentially useful pull requests that will have merge conflicts if white space is adjusted.  Yes, the white space is ugly, but it needs to remain ugly until we have agreement that the source code formatting can be maintained by a program.